bayesian/api/api_v2.py

```diff
@@ -199,6 +197,9 @@ def post():
         response_template: Tuple = namedtuple("response_template", ["message", "status", "headers"])
         input_json: Dict = request.get_json()
         ecosystem: str = input_json.get('ecosystem')
+        user_agent = request.headers.get('User-Agent', None)
+        manifest_hash = str(request.headers.get('manifest_hash', None))
+        request_id = request.headers.get('request_id', None)
```

Comment:
Why are we overloading request header? shouldn't this be optional filed of request payload?

Reply:
@prashbnair

Reply:
I dont think any of these fields are optional. I am now having a doubt if we should capture the path as it reveals some data about the user. I read somewhere that it is not allowed.

Reply:
we can apply the hash at client level. however still SA request has manifest path.